hal/src/rtl872x/usbd_cdc.cpp

```diff
@@ -415,29 +422,34 @@ int CdcClassDriver::startTx() {
         return 0;
     }
 
+    if (holdoff) {
+        stopTxTimer();
+        return startTxTimer();
```

Comment:
What is the use case for `holdoff = true`? This just delays the start of transmission by 1ms?